plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityItem.java

```diff
@@ -59,48 +53,84 @@ private EntityItem(EntityTag entity) {
     EntityTag item;
 
     public ItemTag getItem() {
-        if (item.getBukkitEntity() instanceof Item) {
-            return new ItemTag(((Item) item.getBukkitEntity()).getItemStack());
+        if (isDroppedItem()) {
+            return new ItemTag(getDroppedItem().getItemStack());
         }
-        else if (item.getBukkitEntityType() == EntityType.TRIDENT) {
+        else if (isThrowableProjectile()) {
+            return new ItemTag(((ThrowableProjectile) item.getBukkitEntity()).getItem()); // TODO: 1.15
+        }
+        else if (isTrident()) {
+            // TODO: 1.15: supported by ThrowableProjectile now, remove this part when 1.14 is dropped

```

Comment:
put trident before throwable.
General formatting/ordering rule: things that are version locked, the things meant for earlier versions go first and things locked to later versions go after. This is a compatibility thing that sometimes is explicitly required, and other times just is to have a consistent standard.

Reply:
I see, order should be fixed now

Reply:
it's... not? you... literally didn't change it

Reply:
My bad, only did the mech ;-;
Should be actually done now